.github/workflows/get-environment.yml

```diff
@@ -250,11 +250,46 @@ jobs:
         with:
           script: |
             const { execSync } = require('child_process');
+            const fs = require('fs');
             let version = '';
             if ('${{ inputs.version_file }}'.match(/pom\.xml$/)) {
                 version = execSync(`grep -m 1 "<version>.*</version>" ${{ inputs.version_file }} | sed 's/.*<version>\\(.*\\)<\\/version>.*/\\1/'`).toString().trim();
+
+            } else if ('${{ steps.get_stability.outputs.stability }}' === 'stable') {
+              const { owner, repo } = context.repo;
+
+              // Fetch the most recent tag for plugins
+              const { data: tags } = await github.rest.repos.listTags({
```

Comment:
this call returns the 30 most recent tags ?
otherwise, you should manage pagination

Reply:
since github api list tag returns tag from most recent to older ones, I only keep the single 1 of the single page returned.
this will be the most recent one